packages/components/src/internal/components/user/UserDetailsPanel.tsx

```diff
@@ -135,7 +136,11 @@ export class UserDetailsPanel extends React.PureComponent<Props, State> {
                 this.setState({ userProperties: response.props });
             } else {
                 const response = await api.getUserPropertiesForOther(userId);
-                this.setState({ userProperties: response });
+                if (!Utils.isEmptyObj(response)) {
+                    this.setState({ userProperties: response });
+                } else {
+                    this.setState({ userProperties: { UserId: userId, DisplayName: displayName } });
```

Comment:
Won't these always be undefined here?

Reply:
This is for the case where the grid or display has the userId and display name but the user doesn't have perm to see the full user details because that display user no longer has project perm (i.e. not in core.Users) and the user is not a site admin (i.e. so display user is not in the filtered core.SiteUsers). In that case we just show the info we have/known (i.e. userId and displayName).